styles.css

```diff
@@ -12,6 +12,14 @@ body {
     overflow: auto; /* Enable scrolling within the container */
 }
 
+.settings-icon {
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    font-size: 24px;
+    cursor: pointer;
+}
+
 .tabs {
     display: flex;
     justify-content: flex-start; /* Ensure tabs are aligned to the left */
```

Comment:
While at it, I seem to have figured out the top 
```suggestion
    flex-wrap: wrap; /* Expand vertically instead of horizontally and off-screen */
    justify-content: space-evenly; /* Distribute tabs across the width */
```

And can probably remove the `gap` too, the suggestion won't apply to expanded lines

![Screen Shot 2024-06-17 at 20 32 29](https://github.com/Animeboynz/Mihon-Backup-Viewer/assets/7104931/0c5ab501-9b5c-4103-98ff-41fc4a506e78)